update/provisioner.go

```diff
@@ -100,6 +103,12 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 			errors.New("Must supply an 'username'"))
 	}
 
+	if p.config.UpdateLimit == "" {
+		p.config.UpdateLimit = "1000"
```

Comment:
make this `100` to be consistent with the others places where this number appears